src/main/webapp/dashboard.html

```diff
@@ -32,8 +32,8 @@
     <div id="about-me" class="container tab-pane active"><br>
       <div class="card card-holder col-12 ">
         <div id="about-me-card-body" class="card-body">
-          <img id="user-image" class="card-img-top img-thumbnail w-25 p-2 m-2 
-          mx-auto d-block" src="#" alt="Card image">
+          <img id="user-image" class="card-img-top img-thumbnail p-2 m-2 
+          mx-auto d-block user-img" src="#" alt="Card image">
```

Comment:
I believe wrapped lines should be indented 4 spaces: https://google.github.io/styleguide/htmlcssguide.html

Reply:
```suggestion
              mx-auto d-block user-img" src="#" alt="Card image">
```